packages/core-data/README.md

```diff
@@ -29,9 +29,9 @@ function MyAuthorsList( { authors } ) {
 	);
 }
 
-MyAuthorsList = withSelect( ( select ) => ( {
+const MyAuthorsList = withSelect( ( select ) => ( {
 	authors: select( 'core' ).getAuthors(),
-} ) );
+} ) )( MyAuthorsListBase );
```

Comment:
AFAIK the only change necessary would have been to add `( MyAuthorsList )` invocation. The "Base" separation can make for being more explicit, though maybe an unnecessary / confusion distinction to make for those not already familiar with concepts of presentational and container components.

Reply:
That's true, but other examples use different variable names too, and I think it's less confusing to have different variable names. 🤷‍♂️

I think we should be encouraging this generally… but if I'm off-base we can change it.